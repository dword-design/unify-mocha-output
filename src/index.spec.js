import { endent } from '@dword-design/functions'

import self from '.'

export default [
  () =>
    expect(
      self(endent`
  
    index
      ✓ test1
      ✓ test2
  
  
    2 passing (7ms)
    
  `)
    ).toEqual(endent`
  
    index
      ✓ test1
      ✓ test2


    2 passing
    
  `),
  () =>
    expect(
      self(endent`
  
    index
      √ test1
      √ test2
  
  
    2 passing (7ms)
    
  `)
    ).toEqual(endent`
  
    index
      ✓ test1
      ✓ test2


    2 passing
    
  `),
  () =>
    expect(
      self(endent`
  
    index
      ✓ test1 (1000ms)
      ✓ test2 (2000ms)
  
  
    2 passing
    
  `)
    ).toEqual(endent`
  
    index
      ✓ test1
      ✓ test2


    2 passing
    
  `),
]
