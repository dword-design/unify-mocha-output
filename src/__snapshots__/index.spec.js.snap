// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index 0 1`] = `
"

  index
    ✓ test1
    ✓ test2


  2 passing
"
`;

exports[`index 1 1`] = `
"
index
  ✓ test1
  ✓ test2


2 passing
"
`;

exports[`index 2 1`] = `
"
index
  ✓ test


1 passing
"
`;

exports[`index 3 1`] = `
"
index
  ✓ test1
  ✓ test2


2 passing
"
`;
